flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/internal/options/JdbcOptions.java

```diff
@@ -83,7 +95,7 @@ public boolean equals(Object o) {
 		private String username;
 		private String password;
 		private JdbcDialect dialect;
-
+		private int connectionCheckTimeoutSeconds;
```

Comment:
Would be better to add a default 60 value for this variable. Otherwise, if users are using DataStream API, they will get a zero timeout by default which means never timeout.